libs/librrgraph/src/io/rr_graph_reader.cpp

```diff
@@ -29,6 +30,22 @@
 #    include "mmap_file.h"
 #endif
 
+/**
+ * @brief Parses a line from the RR edge delay override file.
+ *
+ * @details Expected formats:
+ *          edge_id Tdel
+ *          (source_node_id, sink_node_id) Tdel
+ *
+ * @param line The line to parse.
+ * @param overridden_Tdel Parsed override delay.
```

Comment:
Comment that you return overridden_Tdel through this reference parameter.  I'd also probably make it the last argument in that case.

Or just return a tuple.